string coord
string tag
float64 x_coord
float64 y_coord


